advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx

```diff
@@ -11,25 +11,25 @@ EDB Postgres AI is composed of multiple elements which come together to deliver
 
 
 ## [EDB Postgres AI Console](/edb-postgres-ai/console)
-The Console is the gateway to EDB Postgres AI. Starting with a single pane of glass onto all your EDB Postgres AI operations, Console is there to manage your database landscape. It also lets you create and manage your databases, and provides a view of your databases' health and performance. Add to that the ability to bring up Lakehouse Analytics and AI/ML capabilities, and you have the power of ED Postgres AI at your fingertips.
+The Console is the gateway to EDB Postgres AI. Starting with a single pane of glass onto all your EDB Postgres AI operations, Console is there to manage your database landscape. It also lets you create and manage your databases, and provides a view of your databases' health and performance. Add to that the ability to bring up Lakehouse Analytics and AI/ML capabilities, and you have the power of EDB Postgres AI at your fingertips.
 * **[EDB Postgres AI Estate](/edb-postgres-ai/console/estate)**
-  * EDB Postgres AI Cloud Service databases are automatically managed on the Console. The Estate can also monitor the health of AWS RDS and S3 resources and using the Agent, on-premises databases show in the same view.
+  * EDB Postgres AI Cloud Service databases are automatically managed on the Console. The Estate can also monitor the health of AWS RDS and S3 resources and using the Agent, on-premise databases are shown and managed in the same view.
 * **[EDB Postgres AI Agent](/edb-postgres-ai/console/agent)**
-  * Include on premises databases under the Console's single pane of glass view with the Agent. The Agent is part of enabling EDB Postgres AI an unprecedented view of diverse deployments.
+  * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of enabling EDB Postgres AI an unprecedented view of diverse deployments.
```

Comment:
It would read better. 

```suggestion
  * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of giving EDB Postgres AI an unprecedented view of diverse deployments.
```